sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/models/ContainerServiceResourceTypes.java

```diff
@@ -23,7 +32,11 @@ public static ContainerServiceResourceTypes fromString(String name) {
         return fromString(name, ContainerServiceResourceTypes.class);
     }
 
-    /** @return known ExtendedLocationTypes values. */
+    /**
+     * Gets known ExtendedLocationTypes values..
```

Comment:
```suggestion
     * Gets known ExtendedLocationTypes values.
```

Reply:
Fixed by the [commit#ba36bfb](https://github.com/Azure/azure-sdk-for-java/pull/43518/commits/ba36bfbec20ab3096bf1edbdbdc4e5f2d1ade40d)